internal/rbd/rbd_util.go

```diff
@@ -81,6 +81,35 @@ const (
 
 	// clusterNameKey cluster Key, set on RBD image.
 	clusterNameKey = "csi.ceph.com/cluster/name"
+
+	// Qos parameters name of StorageClass.
+	baseReadIOPS            = "BaseReadIOPS"
+	baseWriteIOPS           = "BaseWriteIOPS"
+	baseReadBytesPerSecond  = "BaseReadBytesPerSecond"
+	baseWriteBytesPerSecond = "BaseWriteBytesPerSecond"
+	readIopsPerGB           = "ReadIopsPerGB"
+	writeIopsPerGB          = "WriteIopsPerGB"
+	readBpsPerGB            = "ReadBpsPerGB"
+	writeBpsPerGB           = "WriteBpsPerGB"
+	baseVolSizeBytes        = "BaseVolSizeBytes"
+
+	// Qos type name of rbd image.
+	rbdQosReadIopsLimit  = "rbd_qos_read_iops_limit"
```

Comment:
drop the `rbd` prefix of the variables if you can

Reply:
done